scripts/lib/CIME/namelist.py

```diff
@@ -1919,63 +1965,70 @@ def _parse_name_and_values(self, allow_eof_end=False):
         alternate file format in "groupless" mode.)
 
         >>> _NamelistParser("foo='bar' /")._parse_name_and_values()
-        (u'foo', [u"'bar'"])
+        (u'foo', [u"'bar'"], False)
```

Comment:
Please modify the documentation of this function to indicate the new return value in the tuple.